src/index.html

```diff
@@ -13,9 +13,68 @@
       "
     >
     <title>Product cards</title>
+
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Emblema+One&family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg"
+        alt="image of iMac"
+        class="card_image"
+      >
+        
+      <h1 class="card_product_name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <div class="card_product_code">
+        Product code: 195434
+      </div>
+
+      <div class="card_review">
+      
+        <div class ="stars stars--4">
+          <div class="starts-star"></div>
+          <div class="starts-star"></div>
+          <div class="starts-star"></div>
+          <div class="starts-star"></div>
+          <div class="starts-star"></div>
+        </div>
+
+        <div class="card_review_text">
+          Reviews: 5
+        </div>
+
+      </div>
+
+      <div class="card_price">
+        <div class="card_price_text">Price:</div>
+        <div class="card_price_money">$2,199</div>
+      </div>
+
+      <a 
+        href="#" 
+        class="card_buy_button" 
+        data-qa="hover"
+      >
+      BUY
+      </a>
```

Comment:
you're using `text-transform: uppercase;` so why capitalizing `BUY`?
and a tiny editorial about indentation
```suggestion
      <a 
        href="#" 
        class="card_buy_button" 
        data-qa="hover"
      >
        buy
      </a>
```